deploy/kubernetes/overlays/stable/arm64/kustomization.yaml

```diff
@@ -4,7 +4,7 @@ bases:
   - ../../../base/arm64
 images:
   - name: k8s.gcr.io/provider-aws/aws-ebs-csi-driver
-    newTag: v0.8.1
+    newTag: v0.9.0
```

Comment:
I think we should postpone updating the tags until after we bump the image in gcr. The last two releases I made there was too big of a gap and folks reported issues. https://github.com/kubernetes/k8s.io/pulls?q=is%3Apr+author%3Awongma7+is%3Aclosed

basically process will be
1. merge this without updating tags. tag it with v0.9.0
2. CI builds a "staging" v0.9.0
3. we promote "staging" v0.9.0 to "prod" k8s.gcr.io/provider-aws/aws-ebs-csi-driver
4. update tags like this one

Reply:
actually never mind.
We dont' want to update the helm chart or docs in MASTER until after we bump the image in ecr.

But merging all this  to release-0.9 should be fine.